package/src/renderer/components/image/BoxFit.ts

```diff
@@ -29,19 +29,19 @@ export const rect2rect = (src: IRect, dst: IRect) => {
 
 export const fitRects = (
   fit: Fit,
-  image: IImage,
+  image: IRect,
```

Comment:
Should the parameter be named `rect` as well (instead of image)?

**Also**: This function is one that I happen to use a lot when drawing offscreen, resizing etc. Should we provide it as an export in ´@shopify/react-native-skia`? Maybe a utility namespace-thing?

Reply:
indeed and it is already available via the main export indeed very useful for many cases.